scattr/workflow/rules/qc.smk

```diff
@@ -26,19 +26,33 @@ rule segment_qc:
             else config.get("labelmerge_base_desc"),
             suffix="dseg.nii.gz",
         ),
-        t1w_image=inputs["T1w"].path,
+        # HOW TO PASS APPROPRIATE T1W?
```

Comment:
What is the question here? Do we need to answer it before merging?

Reply:
Ahh, no I fixed this, but forgot to remove the comment - will address in next commit.